sql/core/src/main/scala/org/apache/spark/sql/execution/basicOperators.scala

```diff
@@ -140,8 +140,11 @@ case class TakeOrdered(limit: Int, sortOrder: Seq[SortOrder], child: SparkPlan)
   private def collectData(): Array[Row] = child.execute().map(_.copy()).takeOrdered(limit)(ord)
 
   // TODO: Is this copying for no reason?
```

Comment:
It wasn't relevant before the change as well (that copy call wasn't there anymore). I'll remove it.